products/opensuse/main.pm

```diff
@@ -485,20 +485,8 @@ else {
         load_boot_tests();
         load_zdup_tests();
     }
-    elsif (get_var("BOOT_HDD_IMAGE")) {
-        boot_hdd_image;
-        if (get_var("ISCSI_SERVER")) {
-            set_var('INSTALLONLY', 1);
-            loadtest "iscsi/iscsi_server";
-        }
-        if (get_var("ISCSI_CLIENT")) {
-            set_var('INSTALLONLY', 1);
-            loadtest "iscsi/iscsi_client";
-        }
-        if (get_var("REMOTE_CONTROLLER")) {
```

Comment:
You are removing code for multi-machine tests, but support for it might come soon (with kubic). Then we can use this code to schedule remote installation tests.
If you don't agree then please remove both REMOTE_CONTROLLER and REMOTE_TARGET - one does not make sense without other (below).